telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex

```diff
@@ -0,0 +1,81 @@
+defmodule TelemetryApi.ContractManagers.OperatorStateRetriever do
+  require Logger
+
+  @aligned_config_file System.get_env("ALIGNED_CONFIG_FILE")
+
+  config_file_path =
+    case @aligned_config_file do
+      nil -> raise("ALIGNED_CONFIG_FILE not set in .env")
+      file -> file
+    end
+
+  {status, config_json_string} = File.read(config_file_path)
+
+  case status do
+    :ok ->
+      Logger.debug("Aligned deployment file read successfully")
+
+    :error ->
+      raise("Config file not read successfully")
+  end
+
+  @contract_address Jason.decode!(config_json_string)
+                    |> Map.get("addresses")
+                    |> Map.get("operatorStateRetriever")
+
+  @registry_coordinator_address Jason.decode!(config_json_string)
+                                |> Map.get("addresses")
+                                |> Map.get("registryCoordinator")
+                                |> String.trim_leading("0x")
+                                |> Base.decode16!(case: :mixed)
+
+  use Ethers.Contract,
+    abi_file: "priv/abi/OperatorStateRetriever.json",
+    default_address: @contract_address
+
+  # -------- PUBLIC FUNCTIONS --------
+
+  def get_contract_address() do
+    @contract_address
+  end
+
+  def get_operators() do
+    with {:ok, block_number} = Ethers.current_block_number(),
```

Comment:
In this first version of the telemetry, we want the server not to start in that scenario. Since we don't have retry logic, any operator message will be discarded because we will have no operators registered in our db. We handle this [here](https://github.com/yetanotherco/aligned_layer/pull/1152).

Reply:
I think that is OK, but a clearer error message when not starting up for this reason is a must

Reply:
Added the message `Blockchain not reachable` for this scenario!